module/vpc/main.tf

```diff
@@ -0,0 +1,51 @@
+# Private Network creation
+resource "scaleway_vpc_private_network" "scaleway" {
```

Comment:
It is confusing to have all of them named scaleway

Reply:
Main seems to be a better name if you don't want them to have too much information